src/metabase/lib/breakout.cljc

```diff
@@ -29,3 +30,12 @@
                 expr)]
      (lib.util/update-query-stage query stage-number update :breakout (fn [breakouts]
                                                                         (conj (vec breakouts) expr))))))
+
+(mu/defn breakouts :- [:maybe [:sequential lib.metadata/ColumnMetadata]]
```

Comment:
This stuff was already in `metabase.lib.stage`, but I thought this was a better place for it to live, since we'll probably want to use it elsewhere and `^:export` it at some point as well